Gruntfile.js

```diff
@@ -292,10 +292,9 @@ module.exports = function(grunt) {
     },
 
     shell: {
-      'npm-install': {
-        command: 'node scripts/npm/check-node-modules.js'
+      'install-node-dependencies': {
+        command: 'yarn'
```

Comment:
Won't this take more time that simply comparing two files (s was happening before). I am not very keen on sacrificing speed for _every_ task for this 😕